tests/unit/amo/pages/TestLandingPage.js

```diff
@@ -317,17 +316,15 @@ describe(__filename, () => {
     });
   });
 
-  it('renders the themes shelves with the ADDON_TYPE_THEMES_FILTER filter if static theme is enabled', () => {
-    const fakeConfig = getFakeConfig({ enableFeatureStaticThemes: true });
-
+  it('renders the themes shelves with the ADDON_TYPE_THEMES_FILTER filter', () => {
```

Comment:
Before I believe we added this test just b/c of the config now I am wondering if we need this..? See [PR/commit](https://github.com/mozilla/addons-frontend/commit/cb33d11172242d4a1146c79edad9a6e2a5e2bfe1) that added this. There's nothing for extension so I am just thinking..

if we do keep it I would adjust the name slightly and should we add to the end ~ "...if it's a theme type" and consider adding an extensions test too b/c if we have this for theme should we add something for extensions? I feel like that's what we usually do to show the different types, right?